Description: Fix FTBFS on armhf
Origin: upstream, https://github.com/OpenMathLib/OpenBLAS/commit/8d11e4630c1e92437636d016f35ad48395b62b4f
Reviewed-by: Sébastien Villemot <sebastien@debian.org>
Last-Update: 2025-09-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cpuid.S
+++ b/cpuid.S
@@ -66,5 +66,9 @@ _cpuid:
 
 #endif
 #if defined(__ELF__) && defined(__linux__)
+#if defined(__arm__)
+        .section        .note.GNU-stack,"",%progbits
+#else
         .section        .note.GNU-stack,"",@progbits
 #endif
+#endif
